### PR TITLE
Tutorial for running percona pod on openebs storage in k8s 

### PR DESCRIPTION
**Pull request by ksatchit**
Code changes: 
---------------
Created a tutorial to run percona mysql pod on openebs storage in k8s with steps for load generation (using client container) and stats monitoring (using mayactl)

Changes tested on: 
--------------------
The steps were validated by running them on a local setup with k8s cluster configured on ubuntu 16.04 VMs

### CODE REVIEW

**Comment by kmova** [comment]
Can Step 3 -- be changed to run the mysql client also using a kubectl (may be as an job -- by specifying the timelimit and percona IP as parameters?)

File: k8s/demo/percona/tutorial-running-percona-pod-on-openebs.md [hunk 1]
```diff
@@ -1 +1,128 @@
 # Running percona pod on OpenEBS
+
+This tutorial provides detailed instructions on how to run a percona-mysql application pod on OpenEBS storage in a kubernetes cluster and uses a mysql-client container to generate some load (in the form of insert & select DB queries) in order to illustrate I/O traffic on the storage. 
+
+## Pre-requisites
+
+Pre-requisites include the following: 
+
+- A fully configured kubernetes cluster (versions 1.6.3 & upwards have been tested) with kube master and at least one kube minion. This maybe created on cloud platforms like GKE, on-premise virtual machines (vagrant/VMware/Hyper-V) or bare-metal boxes
```
Comment (AmitKumarDas) [inline k8s/demo/percona/tutorial-running-percona-pod-on-openebs.md]: I would suggest, let us provide the exact versions nos. that were tested.

Comment (ksatchit) [inline k8s/demo/percona/tutorial-running-percona-pod-on-openebs.md]: Sure , Amit. Will do

**Comment by ksatchit** [comment]
Ok Kiran - Will add a kubernetes job for sql load generation. Will close this PR and create a new one with changes.

**Pull request closed without merging**